internal/hcs/system.go

```diff
@@ -95,6 +94,22 @@ func CreateComputeSystem(ctx context.Context, id string, hcsDocumentInterface in
 	return computeSystem, nil
 }
 
+// ModifyServiceSettings modifies settings of the host compute service.
+func ModifyServiceSettings(ctx context.Context, settings hcsschema.ModificationRequest) error {
```

Comment:
This file is all about managing compute systems. Let's move this out to a new file `service.go`.